db/migrate/20150105175908_add_site_id_column_to_reviews_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddSiteIdColumnToReviewsTable < ActiveRecord::Migration
+  def change
+    add_column :reviews, :site_id, :integer, null: false
```

Comment:
generally it's a good idea to index foreign keys

```ruby
add_index :reviews, :site_id
```

This will increase write time, but dramatically improve read time when querying for a given site's reviews